applications/plugins/Geomagic/CMakeLists.txt

```diff
@@ -1,36 +1,49 @@
-cmake_minimum_required(VERSION 2.8.12)
-project(Geomagic)
+cmake_minimum_required(VERSION 3.1)
+project(Geomagic VERSION 0.1)
 
 find_package(SofaHaptics REQUIRED)
-find_package(OpenHaptics)
+find_package(OpenHaptics REQUIRED)
+find_package(SofaOpenglVisual REQUIRED)
```

Comment:
This should be also added to GeomagicConfig.cmake.in in order to auto-trigger dependencies finding when doing `find_package(Geomagic)`.